Documentation/ApiOverview/XmlSitemap/Index.rst

```diff
@@ -172,3 +176,34 @@ The :php:`getItems` method have to return an array with the items for the sitema
 
 The loc element, is the URL of the page that the search engine has to crawl. The lastMod element contains the
 date of the last update of the specific item. This value is a UNIX-timestamp.
+
+
+.. _sitemap-xslFile::
+
+Path to sitemap xslFile
+=======================
+
+.. versionadded:: 10.3
+
+   It is now possible to configure the path to the sitemap xslFile.
+   See changelog :doc:`t3core:Changelog/master/Feature-88147-AddPossibilityToConfigureThePathToSitemapXslFile`
```

Comment:
That will break, as they get moved from `master` to `10.3` folder. Is there a defined process? Once they are cleaned up, all `master` occurrences will be checked and adjusted?

Reply:
You are right. That is actually one of the things I would propose to change: Do not put changelogs in master but already put them in a release folder for the next release (e.g. 10.4 after release 10.3 is out). Can you take that up with the core team for the next release? The current practice will not only break these new references but also all existing URLs which are already indexed in search engines etc. if I am not mistaken. 

I can take care of changing these changelogs links in one batch once 10.3 comes out. Shouldn't be difficult to do a search / replace. I will only add them in the main core references.

It has been pointed out several times, people wish for links to changelog and a clear pointing out when feature was added. I think this is now a good solution.